cli-tests/src/upload.rs

```diff
@@ -127,7 +144,7 @@ async fn upload_bundle() {
     assert_eq!(bundle_meta.envs.get("CI"), Some(&String::from("1")));
     let time_since_upload = chrono::Utc::now()
         - chrono::DateTime::from_timestamp(bundle_meta.upload_time_epoch as i64, 0).unwrap();
-    assert_eq!(time_since_upload.num_minutes(), 0);
+    assert!(time_since_upload.num_minutes() <= 5);
```

Comment:
Nit: This seems to be a fairly popular crate that has an explicit `assert_lt`
https://docs.rs/more-asserts/latest/more_asserts/#

I personally prefer being explicit about expectations in tests since it helps with debugging.

Reply:
No problem. I've been trying to keep things minimal until needed, but I have no issue with adding that. Thanks for doing the research 👍